src/ontology/cl-edit.owl

```diff
@@ -27808,6 +27815,72 @@ AnnotationAssertion(oboInOwl:hasAlternativeId obo:CL_4023154 "CL:0000328")
 AnnotationAssertion(rdfs:label obo:CL_4023154 "myelinating glial cell")
 EquivalentClasses(obo:CL_4023154 ObjectIntersectionOf(obo:CL_0000125 ObjectSomeValuesFrom(obo:BFO_0000051 obo:GO_0043209)))
 
+# Class: obo:CL_4023158 (octopus cell of the mammalian cochlear nucleus)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "WikipediaVersioned:title=Cochlear_nucleus&oldid=1090176664"^^xsd:string) obo:IAO_0000115 obo:CL_4023158 "A neuron in the posterior ventral cochlear nucleus that is distinguished by their long, thick and tentacle-shaped dendrites that typically emanate from one side of the cell body.")
+AnnotationAssertion(obo:IAO_0000116 obo:CL_4023158 "This term is restricted to the mammalian cochlear nucleus to disambiguate for cells of octopus.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4023158 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:11976365"^^xsd:string) Annotation(oboInOwl:hasDbXref "WikipediaVersioned:title=Cochlear_nucleus&oldid=1090176664"^^xsd:string) rdfs:comment obo:CL_4023158 "Octopus cells respond only at the onset of a broad-band simple tonal stimulus and form a pathway to the superior paraolivary nucleus and to the ventral nucleus of the lateral lemniscus.")
+AnnotationAssertion(rdfs:label obo:CL_4023158 "octopus cell of the mammalian cochlear nucleus")
+SubClassOf(obo:CL_4023158 obo:CL_0000540)
+SubClassOf(obo:CL_4023158 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0002831))
+SubClassOf(obo:CL_4023158 ObjectSomeValuesFrom(obo:RO_0013001 obo:UBERON_8440043))
+
+# Class: obo:CL_4023159 (double bouquet cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:15378039"^^xsd:string) obo:IAO_0000115 obo:CL_4023159 "An interneuron that has double bouquet morphology.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4023159 <http://orcid.org/0000-0001-7258-9596>)
+AnnotationAssertion(rdfs:label obo:CL_4023159 "double bouquet cell")
+EquivalentClasses(obo:CL_4023159 ObjectIntersectionOf(obo:CL_0000099 ObjectSomeValuesFrom(obo:RO_0000053 obo:PATO_0070061)))
+
+# Class: obo:CL_4023160 (cartwheel cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "ILX:0102311"^^xsd:string) Annotation(oboInOwl:hasDbXref "WikipediaVersioned:Cartwheel_cell&oldid=1040398094"^^xsd:string) obo:IAO_0000115 obo:CL_4023160 "A neuron of the dorsal cochlear nucleus which spiny dendrites receive input from the axons of granule cells and their axons release GABA and glycine onto cartwheel, pyramidal and giant cell targets.")
```

Comment:
Suggestion for rewording definition:
A neuron of the dorsal cochlear nucleus with spiny dendrites that receive input from the axons of granule cells and with axons that release GABA and glycine onto cartwheel, pyramidal and giant cell targets.